docs/current/cookbook.md

```diff
@@ -370,6 +370,33 @@ The following code listing builds an image using an existing Dockerfile.
 
 [Learn more](./quickstart/429462-build-dockerfile.mdx)
 
+### Build image from Dockerfile using different build context
+
+The following code listing builds an image from a Dockerfile using a build context directory in a different location on the host filesystem.
```

Comment:
Since you use `working directory` on the previous recipe.
I would say

```text
in a different location than the current working directory
```

Reply:
Done